src/features/advanced/cxpo-auto-price.ts

```
@@ -0,0 +1,162 @@
+  container.prepend(div);
+}
+
+function fillQuantity(orders: PrunApi.CXBrokerOrder[], quantityNeeded: number) {
+  const filled = {
+    amount: 0,
+    priceLimit: 0,
+    volume: 0,
+  };
+  for (const order of orders) {
+    // MM orders don't have the amount.
+    const orderAmount = order.amount ?? Infinity;
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)